config/jobs/tensorflow/minigo/minigo.yaml

```diff
@@ -4,23 +4,24 @@ presubmits:
     always_run: true
     branches:
     - master
-    decorate: true
+    labels:
+      preset-service-account: "true"
     spec:
       containers:
-      - image: gcr.io/tensor-go/cc-base:latest
-        command:
-        - /bin/bash
-        - -c
+      - image: gcr.io/minigo-testing/minigo-prow-harness-v2:latest
+        imagePullPolicy: Always
         args:
-        # Copy tensorflow libraries into repo, test with board_size=9, test with board_size=19.
-        - "cp -r /app/cc/tensorflow ./cc/tensorflow && bazel test cc:all --test_output=errors --compilation_mode=dbg --define=board_size=9 && bazel test cc:all --test_output=errors --compilation_mode=dbg --define=board_size=19"
-        securityContext:
-          privileged: true
-        resources:
-          requests:
-            memory: "2Gi"
-      imagePullSecrets:
-      - name: minigo-image-pull
+        - "--job=$(JOB_NAME)"
+        - "--repo=github.com/tensorflow/minigo=$(PULL_REFS)"
+        - --service-account=/etc/service-account/service-account.json
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --scenario=execute
```

Comment:
the non-decorated version of job definition is actually deprecated :-) Besides the image change you might want to make your python test also use  `decorate: true` instead to reduce boilerplates :-)

Reply:
Thanks Sen. I tried to clean it up. Would you mind to take a careful look? I have no idea what I'm doing.